driver/modern_bpf/programs/tail_called/events/custom_logic/drop.bpf.c

```diff
@@ -22,7 +22,7 @@ int BPF_PROG(t1_drop_e)
 
 	/*=============================== COLLECT PARAMETERS ===========================*/
 
-	ringbuf__store_u32(&ringbuf, maps__get_sampling_ratio());
+	ringbuf__store_u32(&ringbuf, 0);
```

Comment:
Same as for the other engines, we can't work without a reliable sampling ratio report (as seen by the driver) :<